code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -0,0 +1,412 @@
+//Reagents produced by metabolising/reacting fermichems inoptimally these specifically are for medicines
+//Inverse = Splitting
+//Invert = Whole conversion
+//Failed = End reaction below purity_min
+
+////START SUBTYPES
+
+///We don't want these to hide - they're helpful!
+/datum/reagent/impurity/healing
+	name = "Healing impure reagent"
+	description = "Not all impure reagents are bad! Sometimes you might want to specifically make these!"
+	chemical_flags = REAGENT_DONOTSPLIT
+	addiction_types = list(/datum/addiction/medicine = 3.5)
+	liver_damage = 0
+
+/datum/reagent/inverse/healing
+	name = "Healing inverse reagent"
+	description = "Not all impure reagents are bad! Sometimes you might want to specifically make these!"
+	chemical_flags = REAGENT_DONOTSPLIT
+	addiction_types = list(/datum/addiction/medicine = 3)
+	tox_damage = 0
+
+//// END SUBTYPES
+
+////////////////////MEDICINES///////////////////////////
+
+//Catch all failed reaction for medicines - supposed to be non punishing
+/datum/reagent/impure/healing/medicine_failure
```

Comment:
I'm not familiar with fermichem; should the typepath here use "impure", or should it use "impurity" or "inverse" like a lot of these other typepaths seem to?

Reply:
Oops, thanks for the catch!